sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactListCriteriaBuilder.java

```diff
@@ -130,6 +141,35 @@
 			contact.get(Contact.EXTERNAL_TOKEN));
 	}
 
+	public List<Selection<?>> getMergeContactIndexSelections(Root<Contact> contact, ContactQueryContext contactQueryContext) {
```

Comment:
this is good, having a specific DTO with the minimal needed selections
but please make sure you reduce the ContactIndexDTO and the selections for the normal grid to the way it was before, there are selections there now that are not used